sdk/Sdk.Generators/Sdk.Generators.csproj

```diff
@@ -20,6 +21,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.11.0" />
+    <PackageReference Include="System.Text.Json" Version="5.0.2" />
```

Comment:
This might need `PrivateAssets="all"`. Same with the `Microsoft.CodeAnalaysis.CSharp` package. Analyzer packages should not cause transitive references.